### DISCUSSION TITLE
What is "Threshold distance"?

### DISCUSSION THREAD

**Discussion by barteis** [osmandapp/OsmAnd#13048] [Q&A] [answered]
When following a GPX, Options/Follow Track/Attach to the roads/Continue leads to a setting called Threshold distance.  I assume this is the minimum distance a road must be to a point in order to qualify, but I'm not sure. 

What's the advantage of setting the distance short versus longer?

**Comment by vshcherb** [comment, marked as answer]
Yes, that's a distance that is allowed for Approximated track to be distant from Original track

**Comment by barteis** [comment]
Thank you.
